examples/stubbing__resources/index.html

```diff
@@ -0,0 +1,34 @@
+<html>
+
+<body>
+  <button id="button">load</button>
+  <div id="img-container">
+  </div>
+
+  <script>
+    function get(url) {
```

Comment:
I think having an Ajax call just to stub it makes the example more confusing than it should be. I would be all for observing the image element and switching the image to fixture.

Reply:
You are right. This was not very helpful and does not make sense at all. Now the image tag is added directly by clicking onto the button